.settings.test.toml

```diff
@@ -45,4 +45,8 @@ basket_url = "http://localhost:4321"
 [playground]
 github_token = "foobar"
 crypt_key = "IXAe2h1MekK4LKysmMvxomja69PT6c20A3nmcDHQ2eQ="
-flag_repo = "flags"
\ No newline at end of file
+flag_repo = "flags"
+
+[ai]
+limit_reset_duration_in_sec = 5
```

Comment:
@fiji-flo Is this supposed to be 5 seconds for testing, or rather 300 seconds = 5 minutes?

Reply:
We wait this duration in the tests, so if it were 5 minutes our tests would take 5 minutes longer to run

Reply:
Yep, that's for tests and supposed to be 5s.